website/src/components/demos/line-demo.js

```diff
@@ -31,10 +31,6 @@ export default class LineDemo extends Component {
     };
   }
 
-  static get viewport() {
-    return INITIAL_VIEW_STATE;
-  }
-
   static get mapStyle() {
     return MAPBOX_STYLES.DARK;
```

Comment:
Does this style have be defined in every demo?

Reply:
Yes.